### ISSUE TITLE
Iceberg Glue view support

### ISSUE THREAD

**Issue by jackye1995** [trinodb/trino#9255]
Currently there are two types of Iceberg views. As a part of supporting Iceberg with Glue, Glue can support the Trino Iceberg view format now, or wait for Iceberg native view support. Details described here: https://docs.google.com/document/d/1uupI7JJHEZIkHufo7sU4Enpwgg-ODCVBE6ocFUVD9oQ/edit

Currently the community consensus seems to be for Trino to migrate once Iceberg native view support is added (proposal 1 in the doc).

This issue is created to collect some data from users to see if we have enough use cases to first add Trino Iceberg view support for Glue. If not, we will throw unsupported for Iceberg Glue users and wait for the Iceberg native view support.

@electrum @phd3 @findepi @ckdarby @dungdm93 @losipiuk

**Comment by findepi** [comment]
> Currently the community consensus seems to be for Trino to migrate once Iceberg native view support is added (proposal 1 in the doc).

There is no such consensus.

> Glue can support the Trino Iceberg view format now

We should do that, in parity with HMS integration.

Note that Hive and Iceberg connectors can share Glue metastore, so Trino views are natural first-class citizen in the ecosystem and the Iceberg connector should support them, even once Iceberg views reach feature-parity with Trino views.

**Comment by jackye1995** [comment]
> There is no such consensus.

Yes sorry I mean on Iceberg side the discussion seems to lead towards Trino migration. It would be great if you can provide more details there.

> We should do that, in parity with HMS integration.

Sounds good, this seems like a good argument to proceed forward.

**Comment by LuPan2015** [comment]
Is it related to this issue?  #9363

**Comment by jayceslesar** [comment]
Is this issue still relevant? If I create a view in spark on a glue catalog for an iceberg table can trino consume that view?
